src/pattern-library/components/patterns/input/TextareaPage.tsx

```diff
@@ -0,0 +1,138 @@
+import { CopyIcon, IconButton, InputGroup, Textarea } from '../../../../';
+import Library from '../../Library';
+
+export default function TextareaPage() {
+  return (
+    <Library.Page
+      title="Textarea"
+      intro={
+        <p>
+          <code>Textarea</code> is a presentational component for styling{' '}
+          <code>textarea</code> elements.
+        </p>
+      }
+    >
+      <Library.Section>
+        <Library.Pattern>
+          <Library.Usage componentName="Textarea" />
+
+          <Library.Example>
+            <Library.Demo title="Basic Textarea" withSource>
+              <div className="w-[350px]">
+                <Textarea
+                  aria-label="Textarea example"
+                  placeholder="Placeholder..."
+                />
+              </div>
+            </Library.Demo>
+          </Library.Example>
+        </Library.Pattern>
+
+        <Library.Pattern title="Working with Textareas">
+          <Library.Example title="Accessibility">
+            <p>
+              Hypothesis does not currently have a design pattern for labeling
+              textareas. However, for accessibility, it is critical that a{' '}
+              <code>Textarea</code> have either an associated <code>label</code>{' '}
+              element or an <code>aria-label</code> attribute.
+            </p>
+
+            <Library.Demo title="Textarea with label" withSource>
+              <div className="w-[350px]">
+                <label htmlFor="textarea-with-label" className="font-semibold">
+                  Label
+                </label>
+                <Textarea
+                  id="textarea-with-label"
+                  placeholder="Placeholder..."
+                />
+              </div>
+            </Library.Demo>
+          </Library.Example>
+          <Library.Example title="Disabled Textareas">
+            <p>
+              <code>Textarea</code>s can be disabled by applying the HTML{' '}
+              <code>disabled</code> attribute.
+            </p>
+            <Library.Demo withSource>
+              <div className="w-[350px]">
+                <Textarea
+                  aria-label="Example textarea"
+                  placeholder="Placeholder..."
+                  disabled
+                />
+              </div>
+            </Library.Demo>
+          </Library.Example>
+          <Library.Demo title="Textarea in an InputGroup" withSource>
+            <div className="w-[380px]">
+              <InputGroup>
+                <Textarea
+                  id="textarea-with-label"
+                  placeholder="Placeholder..."
+                />
+                <IconButton
+                  icon={CopyIcon}
+                  title="Copy to clipboard"
+                  variant="dark"
+                />
+              </InputGroup>
+            </div>
+          </Library.Demo>
+        </Library.Pattern>
```

Comment:
The "Textarea in an InputGroup" example looks weird:

<img width="863" alt="Textarea input group" src="https://github.com/hypothesis/frontend-shared/assets/2458/66f6d059-917b-4eda-b8f6-bcca94cc6b86">

Since we don't have an immediate use case for this, I would just remove this example. If we do want this in future, we'll need to come up with a different design that doesn't look silly. For example we could add the copy button in a footer for the control or as a standalone button in the lower-right corner.

Reply:
Fair enough.